redirects.txt

```diff
@@ -284,4 +284,32 @@ purchase/replenishment/multicompany/setup.rst applications/general/multi_compani
 
 # Redirections introduced in 15.0 :
 
-sales/crm/optimize/outlook_extension.rst productivity/mail_plugins/outlook.rst
\ No newline at end of file
+sales/crm/optimize/outlook_extension.rst productivity/mail_plugins/outlook.rst
+
+developer/reference/addons.rst                                   developer/reference/backend.rst
+developer/reference/addons/actions.rst                           developer/reference/backend/actions.rst
+developer/reference/addons/assets.rst                            developer/reference/backend/assets.rst
+developer/reference/addons/data.rst                              developer/reference/backend/data.rst
+developer/reference/addons/http.rst                              developer/reference/backend/http.rst
+developer/reference/addons/mixins.rst                            developer/reference/backend/mixins.rst
+developer/reference/addons/module.rst                            developer/reference/backend/module.rst
+developer/reference/addons/orm.rst                               developer/reference/backend/orm.rst
+developer/reference/addons/reports.rst                           developer/reference/backend/reports.rst
+developer/reference/addons/security.rst                          developer/reference/backend/security.rst
+developer/reference/addons/testing.rst                           developer/reference/backend/testing.rst
+developer/reference/addons/views.rst                             developer/reference/backend/views.rst
+
+developer/reference/javascript.rst                               developer/reference/frontend.rst
+developer/reference/javascript/framework_overview.rst            developer/reference/frontend/framework_overview.rst
+developer/reference/javascript/javascript_modules.rst            developer/reference/frontend/javascript_modules.rst
+developer/reference/javascript/owl_component_system.rst          developer/reference/frontend/owl_component_system.rst
+developer/reference/javascript/registries.rst                    developer/reference/frontend/registries.rst
+developer/reference/javascript/services.rst                      developer/reference/frontend/services.rst
+developer/reference/javascript/generic_components.rst            developer/reference/frontend/generic_components.rst
+developer/reference/javascript/hooks.rst                         developer/reference/frontend/hooks.rst
+developer/reference/javascript/javascript_cheatsheet.rst         developer/reference/frontend/javascript_cheatsheet.rst
+developer/reference/javascript/javascript_reference.rst          developer/reference/frontend/javascript_reference.rst
+developer/reference/javascript/mobile.rst                        developer/reference/frontend/mobile.rst
+developer/reference/javascript/qweb.rst                          developer/reference/frontend/qweb.rst
+
+developer/reference/backend/assets.rst                           developer/reference/frontend/assets.rst
```

Comment:
I guess I'll have to improve the redirects extension sometime soon to allow globs. (but not today 😅)